src/core/_utils/context.ts

```diff
@@ -0,0 +1,10 @@
+import { URLS_STORAGE_KEY } from 'core/constants';
+
+export const getUrls = () => {
```

Comment:
added some little extra treatments in order to make sure we always return the expected value, even if someone changed the local storage value to a random value trying to break our app  👊